README.md

```diff
@@ -117,6 +117,11 @@ owl0      IEEE 802.11  ESSID:"MyHomeWiFi"
           Power Management:on
 ```
 
+Change wifi list:
```

Comment:
Improve the documentation: SSID Naming Conventions

Reply:
fixed